teps/0003-tekton-catalog-organization.md

```diff
@@ -498,30 +499,42 @@ Infrastructure](https://docs.google.com/document/d/1-czjvjfpuIqYKsfkvZ5RxIbtoFNL
    indicate the name to display in the Tekton Hub, otherwise the name
    will be computed based on the task file name.
 
-4. Resources can include annotation "tekton.dev/tags" which is a comma
+4. Resources must include the annotaion "tekton.dev/category" to indicate the category associated with the resource e.g. git-clone task can have a category as tekton.dev/category: "git"
+
+    1. There should be only one category for a resource. Multiple categories are not allowed.
```

Comment:
TBH I'm sure a single category is a good idea, what's the value it adds compared to a list of tags?

Reply:
So after adding categories in annotation, categories and tags will be independent of each other, resources can be filtered based on only categories and can be searched using tags along with name and displayName